spec/features/user_edits_an_artist_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+feature "user edits an artist", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.